lib/src/result_task.dart

```diff
@@ -0,0 +1,22 @@
+import 'package:meilisearch/src/task.dart';
+
+class ResultTask<T> {
```

Comment:
I changed the name of `ResultTask` to `TaskResult` in the golang SDK not sure it's really relevant but for `TasksQuery` you put Task at the beginning